build_web_compilers/build.yaml

```diff
@@ -12,6 +12,11 @@ builders:
     auto_apply: all_packages
     required_inputs:
       - .dart
+    applies_builders:
+      # We want this to apply for dart2js as well - luckily this builder doesn't
+      # need to be disabled since it's lazy so we can use is as a handle to get
+      # the cleanup builder applied.
```

Comment:
This could instead be set up in its own builder with `auto_appy: all_packages`? It doesn't need to be applied by another builder like this.

Reply:
We didn't add `auto_apply` to `PostProcessBuilders` yet because for our known use cases we had other builders we could attach them too.

I'd be fine adding support though, want me to start working on that?

Reply:
Oh, I forgot about that. I think this is fine for now then, since the same reasoning still applies?